src/Sylius/Bundle/CoreBundle/Resources/config/app/config.yml

```diff
@@ -29,6 +29,12 @@ doctrine:
         entity_managers:
             default:
                 auto_mapping: true
+                mappings:
+                    gedmo_loggable:
+                        type: annotation
+                        prefix: Sylius\Bundle\ResourceBundle\Model
+                        dir: "%kernel.root_dir%/../src/Sylius/Bundle/ResourceBundle/Model"
```

Comment:
Yes :/ Do you think we should provide this path as parameters? Or override this configuration in Sylius-Standard `app/config.yml`?